### DISCUSSION TITLE
Build native lib

### DISCUSSION THREAD

**Discussion by zox47** [flet-dev/flet#4063] [Show and tell]
Flet-Native is a cutting-edge tool designed to simplify the compilation of non-pure Python packages, which may include components written in languages like C, C++, or Rust. 🛠️
https://github.com/zox47/build-lib-native
![Screenshot_2024-09-28-15-06-57-536_com android chrome](https://github.com/user-attachments/assets/3120f25e-b18e-4acc-b27a-e31e77299f17)

**Comment by Muddassir-Farooq-official** [comment]
How to install it and how it works can you share screen recording

**Comment by tanmay-bhatgare** [comment]
@zox47 I am interested in it but it seems this lack enough documentation could you upgrade documentation for easy reproducibility?

**Comment by ndonkoHenri** [comment]
You guys should read this update: https://flet.dev/blog/flet-new-packaging-pre-release

**Reply by Muddassir-Farooq-official** [reply]
I want to confirm one thing in this given link it is cleared that pillow is now directly useable in flet kindly confirm me is it now run on flet web with pyodide or not

**Reply by ndonkoHenri** [reply]
Looking at [this](https://pyodide.org/en/stable/usage/packages-in-pyodide.html#:~:text=3.17.3-,Pillow,-10.2.0) list, it seems to be supported. Give it a try.

**Reply by Muddassir-Farooq-official** [reply]
Is it possible to add functionality in flet to read and display pdf files and csv kindly launch in this new pre release it's too important
